sklearn/feature_selection/tests/test_variance_threshold.py

```diff
@@ -1,33 +1,39 @@
 import numpy as np
 import pytest
-from scipy.sparse import bsr_matrix, csc_matrix, csr_matrix
 
 from sklearn.feature_selection import VarianceThreshold
 from sklearn.utils._testing import assert_array_equal
+from sklearn.utils.fixes import BSR_CONTAINERS, CSC_CONTAINERS, CSR_CONTAINERS
 
 data = [[0, 1, 2, 3, 4], [0, 2, 2, 3, 5], [1, 1, 2, 4, 0]]
 
 data2 = [[-0.13725701]] * 10
 
 
-def test_zero_variance():
+@pytest.mark.parametrize(
+    "constructor", [None, *BSR_CONTAINERS, *CSC_CONTAINERS, *CSR_CONTAINERS]
+)
+def test_zero_variance(constructor):
```

Comment:
Can you please rename the parameter to `container` in this file over here and in the other tests that are changed as well? We are generally using the name `container` so `constructor` sounds confusing and also doesn't seem to be consistent.

Reply:
Sure, thanks for the reminder.